com.microsoft.mrtk.graphicstools.unity/Samples~/Magnifier/Magnifier.meta

```diff
@@ -0,0 +1,8 @@
+fileFormatVersion: 2
```

Comment:
A Magnifier folder in a Magnifier folder might be a little redundant. You could move most of these files up a directory. Please look at the other samples for example structure. 😊